### PR TITLE
Add digital tones

### PR DESCRIPTION
**Pull request by jboolean**
Add the 100 Hz digital tone, which is conspicuously missing from the simulation. 

Well, most of it. Getting the values for DUT1 and leap seconds seems like more effort than it's worth.

Based on the definition on Wikipedia https://en.wikipedia.org/wiki/WWV_(radio_station)#Interpretation

https://github.com/kalafut/wwv/assets/406931/f9a3caea-87a5-4785-a4e6-2122a01700fc

I noticed that the second ticks that are on the deployed site don't seem to play when run locally, maybe the code in production is different from that on master? I don't think that's related to my changes 🤷

### CODE REVIEW

**Comment by kalafut** [comment]
Thanks for the extensive PR! I'll take a look at it soon.

>which is conspicuously missing from the simulation.

I actually recall specifically not doing it because:

- I'd been futzing with V1 forever and just needed to ship it
- My skimming over the encoding was completely wrong, apparently. I'd thought that the encoding was a much higher rate and would need to be in the JS (i.e. couldn't be generated), and that would be at odds with what I built so far. Skimming over your PR I see how it is done and can fit the current model.

> I noticed that the second ticks that are on the deployed site don't seem to play when run locally

I'll take a look. There have been some user-contributed sound updates over the years and something could have gotten out of sync when I was redeploying.

Thanks again!

**Review by kalafut — COMMENTED** [review]
The logic all looks good, but can you remove the changes to `dist/` and `clips/`? I'll be regenerating everything (both JS and MP3) before publishing so these changes would just bloat the repo.

Thanks!

File: src/scheduler.js
Comment (kalafut) [inline src/scheduler.js:1]: I believe there are now some merge conflicts with this. Not your fault... I got around to applying 5 years of JS updates.

**Comment by kalafut** [comment]
This works great. Thank you!

**Comment by jboolean** [comment]
Sorry, I didn't have time to rebase these past few weeks. Thanks for taking care of the conflicts.

**Comment by kalafut** [comment]
It's live now.

**Pull request merged**